.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,13 @@
 
 #### How should this be manually tested?
 
+#### Do the tests adhere to the following recommendations?
```

Comment:
Should we be more specific to which tests we are referring? (Since we just mentioned manual testing above)

```suggestion
#### Do Jest tests follow these best practices?
```